tests/unit/store/traction/pacbio/requests/actions.spec.js

```diff
@@ -1,63 +1,28 @@
-import Response from '@/api/v1/Response'
 import * as Actions from '@/store/traction/pacbio/requests/actions'
-import { Data } from '@support/testHelper'
 import { expect } from 'vitest'
 import { newResponse } from '@/api/v1/ResponseHelper'
+import TractionPacbioSamplesFactory from '@tests/factories/TractionPacbioSamplesFactory.js'
 
 let requests
 
+const tractionPacbioSamplesFactory = TractionPacbioSamplesFactory()
+
 describe('actions', () => {
   beforeEach(() => {
-    requests = new Response(Data.TractionPacbioSamples).deserialize.requests
+    requests = tractionPacbioSamplesFactory.content.data.data
   })
 
   describe('setRequests', () => {
     it('fetches the requets from the service, and commits them', async () => {
       const commit = vi.fn()
       const get = vi.fn()
       const getters = { requestsRequest: { get: get } }
-      const requests = Data.TractionPacbioSamples
-      requests.data.data.splice(2, 11)
+      const requests = tractionPacbioSamplesFactory.responses.fetch
```

Comment:
May be this assignment line can be avoided and use it directly with mockReturnValue.